great_expectations/expectations/core/expect_column_mean_to_be_between.py

```diff
@@ -259,85 +266,60 @@ def validate_configuration(
         self.validate_metric_value_between_configuration(configuration=configuration)
 
     @classmethod
-    def _atomic_prescriptive_template(
+    def _prescriptive_template(
         cls,
-        configuration: Optional[ExpectationConfiguration] = None,
-        result: Optional[ExpectationValidationResult] = None,
-        runtime_configuration: Optional[dict] = None,
-        **kwargs,
-    ):
-        runtime_configuration = runtime_configuration or {}
-        include_column_name = (
-            False if runtime_configuration.get("include_column_name") is False else True
-        )
-        styling = runtime_configuration.get("styling")
-        params = substitute_none_for_missing(
-            configuration.kwargs,
-            [
-                "column",
-                "min_value",
-                "max_value",
-                "row_condition",
-                "condition_parser",
-                "strict_min",
-                "strict_max",
-            ],
+        renderer_configuration: RendererConfiguration,
+    ) -> RendererConfiguration:
+        add_param_args = (
+            ("column", RendererSchemaType.STRING),
+            ("min_value", RendererSchemaType.NUMBER),
+            ("max_value", RendererSchemaType.NUMBER),
+            ("row_condition", RendererSchemaType.STRING),
+            ("condition_parser", RendererSchemaType.STRING),
+            ("strict_min", RendererSchemaType.BOOLEAN),
+            ("strict_max", RendererSchemaType.BOOLEAN),
         )
-        params_with_json_schema = {
-            "column": {"schema": {"type": "string"}, "value": params.get("column")},
-            "min_value": {
-                "schema": {"type": "number"},
-                "value": params.get("min_value"),
-            },
-            "max_value": {
-                "schema": {"type": "number"},
-                "value": params.get("max_value"),
-            },
-            "row_condition": {
-                "schema": {"type": "string"},
-                "value": params.get("row_condition"),
-            },
-            "condition_parser": {
-                "schema": {"type": "string"},
-                "value": params.get("condition_parser"),
-            },
-            "strict_min": {
-                "schema": {"type": "boolean"},
-                "value": params.get("strict_min"),
-            },
-            "strict_max": {
-                "schema": {"type": "boolean"},
-                "value": params.get("strict_max"),
-            },
-        }
+        for name, schema_type in add_param_args:
+            renderer_configuration.add_param(name=name, schema_type=schema_type)
 
-        template_str = ""
-        if (params["min_value"] is None) and (params["max_value"] is None):
+        params: RendererParams = renderer_configuration.params
+
+        if not params.min_value and not params.max_value:
```

Comment:
If `min_value` or `max_value` is numeric (vs a string) then 0 will evaluate to `False` but is still a valid limit.

Reply:
This checks for if `min_value` is a `RendererParam` or `None`. It gets set to `None` if no `value` is passed or found in the `kwargs`. If `min_value` is not `None` and you want to check the `value` you can use `min_value.value`.

Reply:
Setting this to `None` instead of a `RendererParam` gives us the ability to exclude it entirely from the `rendered_content`.